main/EDInitMod.F90

```diff
@@ -436,6 +439,13 @@ subroutine init_cohorts( site_in, patch_in, bc_in)
 
     enddo !numpft
 
+    ! Zero the mass flux pools of the new cohorts
```

Comment:
Are they pools or fluxes that are being zeroed?

Reply:
The fluxes are being zero'd,   ie turnover, net allocated, burned and any others to be introduced.  It also sets the "val0", which remembers the state variable's quantitiy at the beginning of the step, for mass checking later on.